iphone/Classes/PlatformModule.m

```diff
@@ -256,6 +256,21 @@ - (NSString *)identifierForAdvertising
 }
 #endif
 
+- (bool)isRunningOnAppleSilicon
```

Comment:
Maybe name it `isTranslatedBinaryOnAppleSilicon` to indicate that it refers to the Rosetta translation and not just whether the App is running on an Apple Silicon device?